execute/executor_test.go

```diff
@@ -742,7 +743,7 @@ func TestExecutor_Execute(t *testing.T) {
 			// Construct physical query plan
 			plan := plantest.CreatePlanSpec(tc.spec)
 
-			exe := execute.NewExecutor(nil, zaptest.NewLogger(t))
+			exe := execute.NewExecutor(execute.Dependencies{"interpreter": dependencies.NewDefaultDependencies()}, zaptest.NewLogger(t))
```

Comment:
we should be able to change this to executetest.NewTestExecuteDependencies() also